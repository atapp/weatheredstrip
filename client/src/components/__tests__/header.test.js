import React from 'react';
import { render, cleanup } from 'react-testing-library'
import ReactDOM from 'react-dom';
import Header from '../header.react';
import 'jest-dom/extend-expect';
afterEach(cleanup)

it('header renders properly', () => {
  const { container } = render(<Header />)
  const header = container.firstChild
  expect(header).toMatchSnapshot()
});
