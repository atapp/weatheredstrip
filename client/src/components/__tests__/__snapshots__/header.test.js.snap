// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header renders properly 1`] = `
<header
  class="App-header"
>
  <div
    class="App-header-content"
  >
    <div
      class="App-Title"
    >
      Weathered Strip
    </div>
    <form
      class="searchbox"
    >
      <input
        class="searchbox-input"
        placeholder="CYMX CYUL..."
        type="text"
        value=""
      />
      <button
        class="searchbox-button"
        type="submit"
      >
        Get Info
      </button>
    </form>
  </div>
</header>
`;
